doc/api.rst

```diff
@@ -16,6 +16,17 @@ exasol.bucketfs.Bucket
    :undoc-members:
    :show-inheritance:
 
+exasol.bucketfs.path.PathLike
+-----------------------------
+.. autoclass:: exasol.bucketfs._path.PathLike
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+exasol.bucketfs.path.build_path
```

Comment:
The docstring looks slightly odd in the HTML. In the actual docstring, the parameter section consists of 4 parts - common parameters, on-prem, saas, and mounted bucket ones. Sphinx recognizes as parameters only the first section. The rest goes unformatted. I'd want to make all sections look the same, but I don't know how. Any ideas?

Reply:
Maybe let's have a quick video call and see if we can figure that one out together.